src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars__block {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  list-style: none;
+}
+
+.stars--5 .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(1),
+
+.stars--2 .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(1),
+
+.stars--3 .stars__star:nth-child(2),
+
+.stars--3 .stars__star:nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background-image: url(images/star-active.svg);
+}

```

Comment:
refactor this part you just can use something like that `.stars--3 .stars__star:nth-child(-n + 3)` and combine this selectors